### ISSUE TITLE
<Bug report> Wrong avg_pool2d from tensorflow to pytorch

### ISSUE THREAD

**Issue by ain-soph** [microsoft/MMdnn#628]
The bug is for all platforms.

1.
When I convert the inception v3 (version 2015-12-05) (a PB file) from tensorflow to pytorch, it turns out that the average pooling layer is giving the wrong output. 

For example,
`mixed_tower_2_pool = F.avg_pool2d(pool_1, kernel_size=(3, 3), stride=(1, 1), padding=(1,), ceil_mode=False)`

After I debug, I find there should be a parameter in pytorch "`count_include_pad=False`" so that the result is correct. The corner elements are affected since they are divided by 9 rather than 4 (for kernel_size = 3x3). So are the edge elements.

Plz fix this problem. I think it's very easy.
Well, it seems the new inceptionv3 model won't get affected, since they don't use the "same" padding.

2.
Btw, I have to complain about the weight_load process that the converter generates. I hope the transformed "pth" file could be directly loaded.
The present load method is quite strange. I have to use imp and "MainModel".
`MainModel = imp.load_source('MainModel', "torch_inception.py")`
`_model = torch.load("torch_inception.pth")`
And I've got no idea about how to reload the weights after initialization.
I have to `torch.save(_model.state_dict())` , so that I can use it as follows.
I think it's more convenient.

`

class Inception(torch.nn.Module):

    @staticmethod

    def __dense(name, **kwargs):

        layer = nn.Linear(**kwargs)

        return layer

    @staticmethod

    def __batch_normalization(dim, name, **kwargs):
        if   dim == 1:  layer = nn.BatchNorm1d(**kwargs)
        elif dim == 2:  layer = nn.BatchNorm2d(**kwargs)
        elif dim == 3:  layer = nn.BatchNorm3d(**kwargs)
        else:           raise NotImplementedError()
        return layer
    def load_weights(self, weights_loc="torch_inception.pth"):
        self.load_state_dict(torch.load(weights_loc), strict=False)
`

3. I cannot convert the "inception v3" model from the `inNodeName DecodedJpeg`. I can only set it as "Mul", which is the beginning of the convolutional layer. the preprocess can't be converted, which is to decode the jpeg, transfer the tensor to float, and modify the range from (0,255) to (-1,1).

I also cannot convert it using `--dstNodeName aux_logits`

You can easily reproduce this process using the pb file from website: 
 "http://download.tensorflow.org/models/image/imagenet/inception-2015-12-05.tgz"

and command (this works)
`
mmconvert -sf tensorflow -df pytorch -iw classify_image_graph_def.pb --inputShape 299,299,3 --inNodeName Mul --dstNodeName softmax -om pytorch_inception.pth 
`

Platform (like ubuntu 16.04/win10):
Ubuntu 16.04

Python version:
3.7.2

Source framework with version (like Tensorflow 1.4.1 with GPU):
Tensorflow 1.12.0 with GPU

Destination framework with version (like CNTK 2.3 with GPU):
Pytorch

Pre-trained model path (webpath or webdisk path):
Inception v3 2015-12-05  "PB" file
http://download.tensorflow.org/models/image/imagenet/inception-2015-12-05.tgz

**Comment by rainLiuplus** [comment]
Hi @ain-soph,

1. Thanks for the correction, we will fix it soon.

2. Actually, the dumped `.pth` file includes architecture and weights and therefore if you want to do inference you can call `.eval()` directly. However, if you want to load the model via network architecture file and weights, you can use this code: 
`model_converted = imp.load_source('MainModel', converted_file + '.py').KitModel(converted_file + '.npy')` 
This is equivalent to the way you propose, just to be able to call the generated model in another file and you can get the `.npy` file by [this](https://github.com/Microsoft/MMdnn/blob/master/mmdnn/conversion/pytorch/README.md#generate-pytorch-model-from-code-snippet-file-and-weight-file).

3. Currently, MMdnn only supports convert network architecture.
